### DISCUSSION TITLE
Support for PrimeVue 3.50.0

### DISCUSSION THREAD

**Discussion by petr-hybler** [primefaces/community#1518] [PrimeVue]
hello guys, when will be the support for v3.50.0 ? I tried to update the prime-vue package manually and it broke the entire template ... 

It has some nice new features (such as Stepper)

**Comment by tugcekucukoglu** [comment]
The new template versions are released a couple of weeks ago. You could download from store.

**Comment by petr-hybler** [comment]
well I still don't see the stepper implement.. and it's only available in primevue 3.50.0 and your 5.1.0 uses only 3.49.1

**Comment by petr-hybler** [comment]
any update?

**Comment by petr-hybler-ofti** [comment]
hello?

**Comment by petr-hybler-ofti** [comment]
hello?
